documentation/operators/src/legal/landing-pages.md

```diff
@@ -7,3 +7,328 @@ The entire content of this page is under [Creative Commons Attribution 4.0 Inter
 > Nym Node operators are invited to add their legal findings and suggestions directly through [pull requests](add-content.md).
 
 Exit Gateway landing page is a great and transparent way to prevent possible troubles by providing people with basic facts, links to laws and regulations on a given topic as well as operator's contact info. To inspire each other we share some examples how Nym community handled this issue below.
+
+## Avril 14th Exit Gateways
+
+The following snippet it on the 'assets' path under '[http]' section of your config file
+```
+landing_page_assets_path = '<path to your custom html file>'
```

Comment:
For user unique variables we tend to use a convention of capitol letters with no space, ie:
`<PATH_TO_YOUR_CUSTOM_FILE>`. Do you mind to use that one on your guide as well?